applications/GeoMechanicsApplication/custom_processes/apply_write_result_scalar_process.hpp

```diff
@@ -127,21 +121,16 @@ class ApplyWriteScalarProcess : public Process
      */
     void ExecuteFinalizeSolutionStep() override
     {
-        KRATOS_TRY;
-
-        const Variable<double> &var = KratosComponents< Variable<double> >::Get(mVariableName);
-
-        const double Time = mrModelPart.GetProcessInfo()[TIME]/mTimeUnitConverter;
+        KRATOS_TRY
 
         const int nNodes = static_cast<int>(mrModelPart.Nodes().size());
 
-        if (nNodes > 0)
-        {
+        if (nNodes > 0) {
+            const Variable<double> &var = KratosComponents< Variable<double> >::Get(mVariableName);
+            const double Time = mrModelPart.GetProcessInfo()[TIME]/mTimeUnitConverter;
             ModelPart::NodesContainerType::iterator it_begin = mrModelPart.NodesBegin();
 
-            //#pragma omp parallel for
-            for (int i = 0; i<nNodes; i++)
-            {
+            for (int i = 0; i<nNodes; ++i) {
                 ModelPart::NodesContainerType::iterator it = it_begin + i;
 
                 double value = it->FastGetSolutionStepValue(var);
```

Comment:
I would replace `std::endl` with `\n`. The difference is that the former one flushes the buffer which can be significantly slower

You could explicitly flush the file after the loop of you want to be sure